org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/KotlinSMAP.java

```diff
@@ -24,7 +24,7 @@
 /**
  * Parsed representation of SourceDebugExtension attribute.
  */
-final class KotlinSMAP {
+public final class KotlinSMAP {
```

Comment:
Is this still Kotlin specific? Or should we move it to `org.jacoco.core.internal.analysis.SMAP`? This would resolve the dependency from the Analyzer on a Kotlin specific class.

Reply:
> Is this still Kotlin specific? 

@marchof yes - it is still Kotlin specific.